docs/amber_script.md

```diff
@@ -124,33 +130,30 @@ The following commands are all specified within the `PIPELINE` command. If you
 have multiple entry points for a given shader, you'd create multiple pipelines
 each with a different `ENTRY_POINT`.
 
-Bind the entry point to use for a given shader. The default entry point is main.
 
 ```
+  // Bind the entry point to use for a given shader. The default entry point is
+  // main.
```

Comment:
This doesn't quite work.  We can use the same shader module to hold different shader stages.
E.g. the vertex and fragment shaders could be in the same module.  Then this ENTRY_POINT command is ambiguous.

Better to modify the ATTACH command. (see below)

Reply:
Fixed in the Shader CL. ENTRY_POINT is removed and added to ATTACH.